test/fixtures/calendar_events.yml

```diff
@@ -96,18 +96,18 @@ sixteen:
 
 seventeen:
   academic_calendar: lisans_calendar_fall_2017_2018
-  calendar_title: twenty-one
+  calendar_title: twenty_one
```

Comment:
Bu committe yaptığım değişikliği kozmetik bir değişiklik olarak görüp pas geçmeyin. fixture'larda key'lere değişken muamelesi yapıp ona göre isim vermek lazım. Değişken isimlerinde de '-' olmadığı için bunları '_' ile değiştirdim. Neden değişken muamelesi yapmak lazım derseniz: http://serhatdundar.com/blog/rails-instantiated-fixtures